build.gradle

```diff
@@ -66,25 +80,10 @@ spotbugsTest {
     ignoreFailures = true
 }
 
-ext {
-    opendistroVersion = '1.13.0'
-    isSnapshot = "true" == System.getProperty("build.snapshot", "true")
-}
-
-group = "com.amazon.opendistroforelasticsearch"
-version = "${opendistroVersion}.0"
-if (isSnapshot) {
-    version += "-SNAPSHOT"
-}
```

Comment:
Any reason why this was removed? I see all plugins having this

Reply:
This is a mistake, thanks. I meant to move the definition of `group` and `version` so that they would follow the `ext`, but I forgot to add them back. Will fix in new revision.